app/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,146 @@
+    <string name="placeholder_session_abstract">一部がInfiltrate2013とNoSuchCon2013で配信されたこの講演では、（組み込みアプリケーションと携帯端末向け）LinuxとAndroidでの（今年と最後にBlackhatで完売した）Practical ARM Exploitationコースに組み込まれている最近の研究について説明します。 また、これらの技術をデモし、カスタム構築したいくつかのARMハードウェア開発プラットフォームを使用して、それらを発見する方法について説明します。 必要に応じて、共著で2013年10月にリリースされるAndroid Hackers Handbookで説明したように、Androidに関連するARMの悪用についても説明します。</string>
+    <string name="placeholder_session_description">最後に、バスプロトコルの盗聴/リバースエンジニアリング、ハードウェアデバッグを明らかにし、ハードウェア技術を使用して組み込みソフトウェア（ファームウェア）を密かに入手するなど、（上記を容易にするため）最新の関連ハードウェア研究についても説明します。この作業を実行し、Apple MFI iAPデバイス、ならびOEM実装のUSBスタックを使用するマルチメディアデバイスに対してそれらをデプロイするために、使用されるサポートツールと開発された技術を示し、デモします。 （これには、ハードウェアUSBファザーの完全に組み立てられた修正バージョンを販売するhttp://int3.ccの開始に関する経験が簡単に含まれます。）   途中に、（ゼロから）ハードウェアを完全に設計し、ファームウェアを書き、特許を取得した研究者向けのハードウェア脆弱性評価スイスアーミーナイフとして公開しているOspreyと呼ばれる組み込みセキュリティデバイス用のモバイルアプリで学んだ教訓の一部を必然的に共有します。</string>
+    <string name="placeholder_session_speakers">Stephen A. Ridley, Ernest Ridley, Conan Ridley, Bridget Ridley, Frank Ridley</string>
+    <string name="placeholder_session_links">Don\'t Stuff Beans Up Your Nose INT3.CC The Osprey (on INT3)</string>
+    <string name="placeholder_session_online">https://events.ccc.de/congress/2013/Fahrplan/events/5193.html</string>
+    <string name="placeholder_session_language">ja</string>
+    <string name="placeholder_session_day">今日</string>
+    <string name="placeholder_session_start_time">17:15</string>
+    <string name="placeholder_session_duration">240分</string>
+    <string name="placeholder_session_room">Village:Königlich Bayerisches Amtsvillage</string>
+    <string name="placeholder_schedule_time">09:15</string>
+    <string name="placeholder_alarm_time">2013年12月28日　午後5時10分</string>
+    <string name="placeholder_session_date">2013年12月28日　午後5時15分</string>
+    <string name="placeholder_session_location">本当に広い部屋</string>
+    <string name="placeholder_session_id">ID: 5193</string>
+    <string name="placeholder_day_separator_title">2日目</string>
+    <string name="preference_title_auto_update_enabled">自動更新</string>
+    <string name="preference_dialog_title_alarm_time">デフォルトのアラーム時間を選択します</string>
+    <string name="preference_title_alarm_time">デフォルトのアラーム時間</string>
+    <string name="preference_title_alarm_tone">アラーム音</string>
+    <string name="preference_title_alternative_highlighting_enabled">代替の強調表示</string>
+    <string name="preference_title_alternative_schedule_url">代替のスケジュールURL</string>
+    <string name="preference_title_app_notification_settings">通知をカスタマイズ</string>
+    <string name="preference_title_insistent_alarms_enabled">しつこいアラーム</string>
+    <string name="preference_summary_engelsystem_json_export_url">"         <![CDATA[シフトを確認するには、ここに<u>個人</ u>のURLを入力してください。<br/><br/>         1. あなたのEngelsystemアカウントにログインします。<br/>         2.プロフィールページの<u> JSON出力</ u>ボタンの後ろにあるURLをコピーします.<br/>         3. URLをここに貼ります。<br/><br/>         メイン会議日の前後のシフトは<u>表示されません</ u>。]]>     "</string>
```

Comment:
Please only define the `preference_` strings in the `preferences.xml` file. Currently, the build fails because they are defined twice.